server/internal/usecase/repo/request.go

```diff
@@ -0,0 +1,24 @@
+package repo
+
+import (
+	"context"
+
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/request"
+	"github.com/reearth/reearthx/usecasex"
+)
+
+type RequestFilter struct {
+	State      *request.State
+	Keyword    *string
+	Pagination *usecasex.Pagination
+}
+
+type Request interface {
+	Filtered(ProjectFilter) Request
+	FindByProject(context.Context, id.ProjectID, RequestFilter) ([]*request.Request, *usecasex.PageInfo, error)
```

Comment:
```suggestion
}

type Request interface {
	Filtered(ProjectFilter) Request
	FindByProject(context.Context, id.ProjectID, RequestFilter, *usecasex.Pagination) ([]*request.Request, *usecasex.PageInfo, error)
```

I think pagination is not a part of "filter".